cernopendata/modules/theme/assets/semantic-ui/js/search/components/DefaultListItem.js

```diff
@@ -54,15 +54,15 @@ const CODefaultListItem = ({ result: { metadata, id }, index }) => {
             metadata.type.secondary.map((subtype) => (
               <Label
                 key={subtype}
-                className="badge badge-subtype"
+                className="tiny teal"
                 href={`/search?f=subtype:${subtype}`}
```

Comment:
Shouldn't we also amend search URLs as part of the PR?

Note that old URLs might be supported as part of #3030, but it would be good if the site itself does not generate old URLs, only new ones.

Example: when testing this PR, the links work but the search UI facet checkboxes are not always ticked, try to click on "Dataset" badge as opposed to "Simulated" badge for example:

![simulated](https://user-images.githubusercontent.com/517546/107488482-b5377680-6b87-11eb-8d40-70297fbd1f17.png)

The number of results is well 3437, but the facet checkbox is not selected. (Unlike for "Dataset" badge, which is top level, and where this works.)